setup.cfg

```diff
@@ -1,3 +1,3 @@
 [pycodestyle]
-ignore = E402
-max-line-length = 80
+ignore = E121, E123, E126, E226, E24, E704, W503, E402
+max-line-length = 120
```

Comment:
We should stick with 80, because that's the most accepted and widely used value in projects.